onnxruntime.spec

```diff
@@ -21,6 +21,7 @@ rm -rf ../build; mkdir ../build; cd ../build
 USE_CUDA=OFF
 if [ "%{cuda_gcc_support}" = "true" ] ; then
   USE_CUDA=ON
+  CUDA_FLAGS=$(echo "-DTHRUST_IGNORE_DEPRECATED_API -DCUB_IGNORE_DEPRECATED_API %{nvcc_cuda_flags}" | tr '\n' ' ')
```

Comment:
This is actually includes too many flags, some are already set by `CMAKE_CUDA_ARCHITECTURES=...` and `CMAKE_CUDA_RUNTIME_LIBRARY=Shared`.